tools/sdk/services/data.go

```diff
@@ -121,3 +121,16 @@ func GetResourceManagerServicesByName(client resourcemanager.Client, servicesToL
 		Services: resourceManagerServices,
 	}, nil
 }
+
+// GetResourceManagerCommonTypes returns the specified Services from the Data API
+func GetResourceManagerCommonTypes(client resourcemanager.Client) (*ResourceManagerCommonTypes, error) {
```

Comment:
(fwiw once the v2 Data API is merged we'll update the SDK to remove the `resourcemanager` references, presumably having a `data-api-sdk` folder or similar?)